Backpack/src/main/java/net/skyscanner/backpack/util/ItemHolder.kt

```diff
@@ -0,0 +1,39 @@
+package net.skyscanner.backpack.util
+
+import android.content.Context
+import android.content.res.Resources
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.IdRes
+import androidx.annotation.LayoutRes
+import androidx.recyclerview.widget.RecyclerView
+
+internal abstract class ItemHolder<T>(
```

Comment:
This seems a bit too specific to the BarChart component to be here, at lest for now. Can we move this inside the bar chart package?

Reply:
I want this to be used in every Backpack components using RecyclerView. This can be moved for now and then re-moved to util package when it's needed. Do you want me to do that?

Reply:
I think it's better to wait for abstractions to be needed before we generalise them like this, we will see it better when we have an actual need for this in more components, so I'd keep in the chart component for now. Up to you